DataFormats/TotemDigi/src/classes_def.xml

```diff
@@ -15,7 +15,7 @@
   <class name="edm::Wrapper<TotemTriggerCounters>"/>
 
   <class name="TotemVFATStatus" ClassVersion="2">
-    <version ClassVersion="2" checksum="3003040825"/>
+    <version ClassVersion="2" checksum="1439138260"/>
```

Comment:
given that some other changes appear necessary https://github.com/cms-sw/cmssw/pull/14615/files#r65102149
please update ClassVersion here to 3:

```xml
  <class name="TotemVFATStatus" ClassVersion="3">
    <version ClassVersion="2" checksum="3003040825"/>
    <version ClassVersion="3" checksum="1439138260"/>
```